sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuBroadcastToRowExec.scala

```diff
@@ -62,12 +62,13 @@ case class GpuBroadcastToRowExec(
     SQLExecution.withThreadLocalCaptured[Broadcast[Any]](
       session, GpuBroadcastToRowExec.executionContext) {
       val broadcastBatch = child.executeBroadcast[Any]()
-      val result: Any = broadcastBatch.value match {
+      val rows: Array[InternalRow] = broadcastBatch.value match {
```

Comment:
Fascinating. Minor nit: a comment here would be nice to explain why the specific type here is required.